### PR TITLE
ST-1437 -Added publish flag in Event, for WA configuration in Special Tribunals

### PR DESCRIPTION
**Pull request by ShwetaTandel-hmcts**
### Change description ###
ST-1437 - Added publish flag in Event, for WA configuration in Special Tribunals

### CODE REVIEW

File: ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java [hunk 1]
```diff
@@ -116,6 +117,17 @@ public EventBuilder<T, R, S> showSummary() {
       return this;
     }
 
+
+    public EventBuilder<T, R, S> publish(boolean publish) {
```
Comment (linusnorton) [inline ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java:121]: Is there any way we can make it clearer what this flag is? What does publish do?

Comment (linusnorton) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java:121]: Great, should we call the field and method `publishToCamunda`?

Comment (ShwetaTandel-hmcts) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java:121]: I am afraid we cannot, we need to use Publish as the name in order that the event gets picked up to be published by the wa-case-event-handler. I checked with other teams likes SCSS, IA & ET , they are all using the same flag name.

Comment (linusnorton) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java:121]: That's not true. The field and method can be publishToCamunda. The JSON output will be Publish

Comment (ShwetaTandel-hmcts) [reply ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java:121]: Oh yes, you meant just the variable name. Yes that can be done, I'll do the change.

**Review by linusnorton — APPROVED** [review]

**Pull request merged**